libs/containers/Interaction.lua

```diff
@@ -65,9 +65,12 @@ function Interaction:__init(data, parent)
     -- Last resort, request the channel object from the API if wasn't cached
     if not self._channel then
       local channel = self._api:getChannel(channelId)
-      if not channel then goto skip end -- somehow channel not available
+      if not channel then goto skip end
 
       local guild = channel.guild and parent._guilds:_insert(channel.guild)
+      if (not guild) and (channel.guild_id) then
```

Comment:
- This should use the style used in the rest of the if statements in the repo, no optional parentheses.
- why is it checking for `channel.guild_id`, is there any instance when the channel is available but a guild_id is not provided?

Can't this simply be `local guild = channel.guild and parent._guilds:_insert(channel.guild) or parent:getGuild(channel.guild_id)`